Sources/SimultaneouslyScrollView/Extensions/MulticastScrollViewDelegate.swift

```diff
@@ -0,0 +1,107 @@
+import UIKit;
+
+class MulticastScrollViewDelegate: NSObject, UIScrollViewDelegate {
```

Comment:
Please explicitly add the `internal` modifier.

```suggestion
internal class MulticastScrollViewDelegate: NSObject, UIScrollViewDelegate {
```